src/monster.cpp

```diff
@@ -605,10 +612,14 @@ void Monster::goToFollowCreature()
 			startAutoWalk();
 		}
 	} else {
-		updateFollowCreaturePath(fpp);
```

Comment:
use updateFollowCreaturePath
its the exact duplicate of the code you pasted here, so there is no reason to duplicate the code again